src/lib.rs

```diff
@@ -112,6 +112,15 @@
 //! the number of sinks that can be created (except for the fact that creating too many will slow
 //! down your program).
 //!
+//! 
+//! ## Features 
+//!
+//! Rodio provides several optional features that are guarded with feature gates.
```

Comment:
This is the only <thing> thats not a decoder that lives behind a feature gate. So its not `several` right?

Reply:
I dunno I expected all the features to eventually be in here, I'll change the language to make it singular.

Reply:
You where right :laughing:, rodio got optional support for tracing (logging library) so we need that description. I'm copying it to main now. You can add the synthesizer note to that.

Reply:
I've fixed the documentation here so both flags are in the same header block.